ts/defs.bzl

```diff
@@ -291,6 +291,9 @@ def ts_project(
             tsconfig = "tsconfig.json"
 
     if type(tsconfig) == type(dict()):
+        # Output declarations by default if unspecified and not extending a config which may specify
+        declaration = False if declaration == False or extends else True
```

Comment:
had to read four times to understand the outcome 🗡️

Reply:
Have a recommendation or preference to use instead?